packages/IMAPClient-Core.package/ICAddressBook.class/instance/update.with.andWith.andWith..st

```diff
@@ -0,0 +1,11 @@
+accessing
+update: anAddressBookEntry with: newFirstName andWith: newLastName andWith: newEmail
+
+	| index entryToUpdate |
+	index := self entries find: anAddressBookEntry.
```

Comment:
why do you search for it in the collection and do not update directly with the key

Reply:
Because we do not have a key, we get an object and have to find it again.
The only key is the index.